dev/NavigationView/NavigationView.cpp

```diff
@@ -1115,6 +1119,51 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
+{
+    if (PaneDisplayMode() == winrt::NavigationViewPaneDisplayMode::Top
+        || DisplayMode() == winrt::NavigationViewDisplayMode::Minimal)
+    {
+        if (auto backButton = m_backButton.try_as<winrt::Button>())
+        {
+            backButton.Width(40);
+        }
+        if (auto paneToggleButton = m_paneToggleButton.try_as<winrt::Button>())
```

Comment:
>m_paneToggleButton.try_as<winrt::Button>() [](start = 36, length = 42)

m_paneToggleButton is a tracker ref to a button, you should just call .get() instead of casting

Reply:
Oh yes, that does work. My question is, why are we using try_as in other places even when we are using tracker_ref there too?

Reply:
hmm, do you have a specific example? when the type you want is not the same as the type held by the tracker ref you need to perform a cast on that type.  In this case the types match so it isn't needed.

Reply:
Oh, my mistake, didn't notice that those cases had a mismatch between ref type and desired type.